src/Chapter1.hs

```diff
@@ -632,7 +650,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Integral a => a -> a
+sumLast2 n = (+) l s
+  where
+    n' = abs n
+    l  = lastDigit n'
+    s  = div (mod n' 100) 10
```

Comment:
Nice idea on reusing the `lastDigit` function here! Though, you can reuse it one more time here.

Alternatively, you can implement this function by using `divMod`: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (mod x 100) 10
```

You can see how we could pattern match on the pair 🙂

Reply:
Nice!